ark-blockstore-example/noop-plugin.go

```diff
@@ -41,17 +40,13 @@ func NewNoOpBlockStore() *NoOpBlockStore {
 	return &NoOpBlockStore{logger: plugin.NewLogger()}
 }
 
-func (p *NoOpBlockStore) log(msg string, args ...interface{}) {
+func (p *NoOpBlockStore) logger.Infof(msg string, args ...interface{}) {
```

Comment:
this isn't quite what I had in mind.  You'd want something like:

```
type NoOpBlockStore struct {
    ...
    logrus.FieldLogger
    ....
}
```

Then, you can remove this function entirely, while still being able to use the syntactically simpler `p.Infof(...)` anywhere you need to log!